factory/config.go

```diff
@@ -28,6 +29,18 @@ type Cluster struct {
 	NumServers   int
 	NumAgents    int
 	Config       clusterConfig
+	Infra
+}
+
+type Infra struct {
```

Comment:
This struct can be renamed to `EC2Props`.. just a suggestion...

Reply:
done